@navikt/core/css/darkside/internalheader.darkside.css

```diff
@@ -13,52 +12,51 @@
     border: solid 1px ButtonText;
     color: ButtonText;
   }
-
-  .navds-internalheader a:focus-visible {
-    box-shadow: none;
-    outline: 2px solid highlight;
-    outline-offset: 2px;
-  }
-
-  .navds-internalheader .navds-internalheader__button:focus-visible {
-    box-shadow: none;
-    outline: 2px solid highlight;
-    outline-offset: 2px;
-  }
 }
 
 .navds-internalheader__title {
   border: none;
   overflow: visible;
   background: transparent;
   font: inherit;
-  font-weight: var(--a-font-weight-bold);
+  font-weight: var(--ax-font-weight-bold);
   line-height: normal;
   text-align: left;
-  padding: 0 var(--a-spacing-5) 0 var(--a-spacing-6);
-  border-right: 1px solid var(--ac-internalheader-divider, var(--a-gray-600));
+  padding-block: 0;
+  padding-inline: var(--ax-spacing-6) var(--ax-spacing-5);
+  border-right: 1px solid var(--ax-border-subtleA);
   display: flex;
   align-items: center;
   text-decoration: none;
   margin: 0;
-  color: var(--ac-internalheader-text, var(--a-text-on-inverted));
-}
+  color: var(--ax-text-default);
 
-button.navds-internalheader__title {
-  cursor: pointer;
+  &:is(button) {
```

Comment:
Seems unnecessary to use `:is()` here, but maybe you have to? Could this work? `&button {` Or maybe this is harder to read?

Reply:
Seems like i was baited by the preview since it shows this syntax:
![Screenshot 2024-11-21 at 13 56 01](https://github.com/user-attachments/assets/e5adf702-1dae-4084-8721-9452a4481fb9)
[But when tested here](https://lightningcss.dev/playground#%7B%22minify%22%3Afalse%2C%22customMedia%22%3Atrue%2C%22cssModules%22%3Afalse%2C%22analyzeDependencies%22%3Afalse%2C%22targets%22%3A%7B%22chrome%22%3A6225920%7D%2C%22include%22%3A0%2C%22exclude%22%3A0%2C%22source%22%3A%22.navds-internalheader__title%20%7B%5Cn%20%20%26%3Ais(button)%20%7B%5Cn%20%20%20%20cursor%3A%20pointer%3B%5Cn%20%20%7D%5Cn%7D%5Cn%5Cn.test%20%7B%5Cn%20%20%26button%20%7B%5Cn%20%20%20%20cursor%3A%20pointer%3B%5Cn%20%20%7D%5Cn%7D%22%2C%22visitorEnabled%22%3Afalse%2C%22visitor%22%3A%22%7B%5Cn%20%20Color(color)%20%7B%5Cn%20%20%20%20if%20(color.type%20%3D%3D%3D%20'rgb')%20%7B%5Cn%20%20%20%20%20%20color.g%20%3D%200%3B%5Cn%20%20%20%20%20%20return%20color%3B%5Cn%20%20%20%20%7D%5Cn%20%20%7D%5Cn%7D%22%2C%22unusedSymbols%22%3A%5B%5D%2C%22version%22%3A%22local%22%7D) it works as expected. I can change it 👍

Reply:
Looks like this doesn't work after all. I guess it's a Lightningcss feature?

Reply:
😞